docs/prerequisites/feature-flagging/randomization-sdk/server-sdks/java.md

```diff
@@ -0,0 +1,99 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# JAVA
+
+Eppo's JAVA SDK is open source:
+- [GitHub repository](https://github.com/Eppo-exp/java-server-sdk)
+- [Package](https://s01.oss.sonatype.org/#nexus-search;quick~eppo-server-sdk)
+
+### 1. Install the SDK
+In your pom.xml, add the SDK package as a dependency:
+
+```
+<dependency>
+  <groupId>cloud.eppo</groupId>
+  <artifactId>eppo-server-sdk</artifactId>
+  <version>1.0.0</version>
+</dependency>
+```
+
+### 2. Define an Assignment Logger
+
+The SDK requires an assignment logger to be passed on initialization. The SDK invokes the logger to capture assignment data whenever a variation is assigned. The below code example shows how to integrate the SDK with [Segment](https://segment.com/docs/) for logging events. You could also use your own logging system; the only requirement is that the SDK receives a `logAssignment` function.
```

Comment:
> The below code example shows how to integrate the SDK with [Segment](https://segment.com/docs/) for logging events.

The code example below does not show how to use it with Segment. Please either remove that line or include the Segment code in the example.

Reply:
Fixed the statement. Removed the line regarding segment integration